cmake/modules/patch-dpdk-conf.sh

```diff
@@ -5,7 +5,8 @@ setconf() {
     local key=$1
     local val=$2
     if grep -q ^$key= ${conf}; then
-        sed -i "s:^$key=.*$:$key=$val:g" ${conf}
+        sed -e "s:^$key=.*$:$key=$val:g" ${conf} > ${conf}.new
```

Comment:
`sed -i -e "..."` or `sed -i '' -e "..."` could also work on freebsd, and no need to `mv ...` line then.
But just ignore my comment if you like your variant more.

Reply:
i like yours more.